libs/langchain/langchain/llms/bedrock.py

```diff
@@ -229,6 +236,16 @@ def validate_environment(cls, values: Dict) -> Dict:
 
         return values
 
+    @validator("streaming")
+    def check_flag(cls, value):
+        if value:
+            warnings.warn(
+                "The 'streaming' attribute is set to True. Token usage statistics will "
+                "not be available.",
+                BedrockStreamingWarning,
+            )
```

Comment:
I am not sure if a warning is warranted here, since there are going to be a lot of users who will be using streaming, and this will also not be an error from the user to set this flag. If we really want to do something, it might be useful to just use the logger and log this info. It might also be useful to just note this in the docs for the callback.

Reply:
Is there a reason we can't expose this via streaming? The response metrics are included on the end token.